.azure-pipelines/live-functional.yml

```diff
@@ -0,0 +1,127 @@
+
+    jobs:
+    - job: 'azure_nightly_test_run'
+      displayName: 'Azure Nightly Test Run'
+      timeoutInMinutes: 0
+      cancelTimeoutInMinutes: 0
+
+      pool:
+        vmImage: 'Ubuntu-16.04'
+
+      steps:
+        - checkout: self
+          fetchDepth: 1
+
+        - task: UsePythonVersion@0
+          inputs:
+            versionSpec: '3.7'
+            architecture: 'x64'
+
+        - script: python -m pip install --upgrade pip && pip install . && pip install -r requirements-dev.txt
+          displayName: "Install Dependencies"
+
+        - script: az login --service-principal -u ${AZURE_CLIENT_ID} -p ${AZURE_CLIENT_SECRET} --tenant ${AZURE_TENANT_ID} && az account set -s ${AZURE_SUBSCRIPTION_ID}
+          displayName: "Login to Azure"
+          env:
+            AZURE_CLIENT_ID: $(azure-client-id)
+            AZURE_CLIENT_SECRET: $(azure-client-secret)
+            AZURE_SUBSCRIPTION_ID: $(azure-subscription-id)
+            AZURE_TENANT_ID: $(azure-tenant-id)
+
+        # Skip some resources due to some restrictions with their provisioning using Service Principal account
+        # We maintain those resources in the test subscription available for the tests
+        - script: tools/c7n_azure/tests/templates/provision.sh --skip keyvault cost-management-export containerservice databricks
+          displayName: "Provision Azure Resources"
+          env:
+            AZURE_CLIENT_ID: $(azure-client-id)
+            AZURE_CLIENT_SECRET: $(azure-client-secret)
+
+        - script: C7N_TEST_RUN=true C7N_FUNCTIONAL=yes pytest -v -m "not skiplive" tools/c7n_azure/tests
+          displayName: "Run Azure tests without cassettes"
+
+        - script: tools/c7n_azure/tests/templates/cleanup.sh --skip keyvault cost-management-export containerservice databricks
+          displayName: "Cleanup Azure Resources"
+          condition: always()
+
+    - job: 'azure_nightly_functions_test'
+      displayName: 'Azure Functions Nightly Test'
+      timeoutInMinutes: 0
+      cancelTimeoutInMinutes: 0
+
+      pool:
+        vmImage: 'Ubuntu-16.04'
+
+      steps:
+        - checkout: self
+          fetchDepth: 1
+
+        - task: UsePythonVersion@0
+          inputs:
+            versionSpec: '3.7'
+            architecture: 'x64'
+
+        - script: python -m pip install --upgrade pip && pip install . && pip install -r requirements-dev.txt
+          displayName: "Install Dependencies"
+
+        - script: ./test_functions.sh
+          workingDirectory: tools/c7n_azure/tests/azure-functions
+          displayName: "Run Azure Functions Test"
+          env:
+            AZURE_CLIENT_ID: $(azure-client-id)
+            AZURE_CLIENT_SECRET: $(azure-client-secret)
+            AZURE_SUBSCRIPTION_ID: $(azure-subscription-id)
+            AZURE_TENANT_ID: $(azure-tenant-id)
+
+  # If all functional test stages complete successfully then
+  # push a docker image and tag `nightly`.
+  - stage: post_success

```

Comment:
The new bits are from here down...  Github isn't displaying the rename/diff properly.